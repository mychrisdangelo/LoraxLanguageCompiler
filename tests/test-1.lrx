
int a;//0

int chris(int d){
}


int main()
{
}


