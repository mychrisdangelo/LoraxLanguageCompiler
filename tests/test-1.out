#include "lrxlib.h"
int a_0;

int main();
int chris(int);

int main()
{
int __tmp_int_1 = 0;

return __tmp_int_1;

}
int chris(int d_1)
{
int __tmp_int_0 = 0;

return __tmp_int_0;

}
